kvbc/src/direct_kv_storage_factory.cpp

```diff
@@ -93,7 +93,7 @@ std::unique_ptr<storage::IMetadataKeyManipulator> S3StorageFactory::newMetadataK
 }
 
 std::unique_ptr<storage::ISTKeyManipulator> S3StorageFactory::newSTKeyManipulator() const {
-  return std::make_unique<storage::v1DirectKeyValue::STKeyManipulator>();
+  return std::make_unique<storage::s3::STKeyManipulator>(s3Conf_.pathPrefix);
```

Comment:
If I understand correctly, this is how we forward the ST metadata to S3.

Reply:
this is how ST metadata keys are generated for S3